src/outlier_detection/auto_outliers.py

```diff
@@ -67,9 +68,10 @@ def filter_valid(df: pd.DataFrame, value_col: str) -> pd.DataFrame:
         )
         raise ValueError
     status_cond = df.statusencoded.isin(["210", "211"])
-    positive_cond = df[value_col] > 0
+    pos_cond = df[value_col] > 0
+    ins_cond = df["instance"] == 0
```

Comment:
Why didn't we need the `ins_cond` before?

Reply:
because we hadn't done short to longform conversion, so it was always instance 0

Reply:
(we don't need it for the bugfix, but I'll leave it in for the future)